### ISSUE TITLE
0.13: [Bug]: Cli command "garden migrate" gives error

### ISSUE THREAD

**Issue by fabianboerner** [garden-io/garden#4450]
### Current Behavior

garden migrate garden.yml 
Cannot read properties of undefined (reading 'split')

### Expected behavior

Should give a understandable error message if something is missing or not correct set up

### Reproducible example

```
kind: Module

type: container

name: campaign-api

build:
  timeout: 4000
  targetImage: ${var.targetImage || "development"}
buildArgs:
  npm_token: ${var.npm_token}
  app_port: ${var.app_port}
variables:
  $merge: ${var.commonEnvVars}
  app_port: 80
  hostname: ${var.hostnames.campaign_api}
services:
  - name: my-api
    env:
      BASE_DOMAIN: ${var.base_domain}
    timeout: 4000
    ports:
      - name: http
        containerPort: ${var.app_port}
        servicePort: 80
    ingresses:
      - path: /
        hostname: ${var.hostname}
        port: http
    volumes:
      - name: my-volume
        containerPath: /usr/local/share/.cache/yarn
sync:
  paths:
    - target: /app

dockerfile: ${var.dockerfileName}
```

### Your environment

* **OS:** Ubuntu 22.04

`garden version` 0.13.0

**Comment by remotezygote** [comment]
Same here. More context:

```
TypeError: Cannot read properties of undefined (reading 'split')
    at findProjectConfig (/snapshot/project/tmp/pkg/cli/node_modules/@garden-io/core/src/config/base.ts:443:23)
    at MigrateCommand.action (/snapshot/project/tmp/pkg/cli/node_modules/@garden-io/core/src/commands/migrate.ts:74:50)
    at MigrateCommand.run (/snapshot/project/tmp/pkg/cli/node_modules/@garden-io/core/src/commands/base.ts:295:29)
    at GardenCli.runCommand (/snapshot/project/tmp/pkg/cli/node_modules/@garden-io/core/src/cli/cli.ts:340:16)
    at GardenCli.run (/snapshot/project/tmp/pkg/cli/node_modules/@garden-io/core/src/cli/cli.ts:520:26)
    at Object.runCli (/snapshot/project/tmp/pkg/cli/src/cli.ts:42:14)
```

**Comment by stefreak** [comment]
Successfully reproduced this issue even with the most simple Garden project:

```
kind: Project
apiVersion: garden.io/v0
name: repro
environments:
 - name: default
providers:
 - name: local-kubernetes
```

```
% garden migrate
Cannot read properties of undefined (reading 'split')

See .garden/error.log for detailed error message
```

**Comment by stefreak** [comment]
@fabianboerner @remotezygote Good news here is that, according to the help, this command helps migrate from garden 0.11 => 0.12 configs. For 0.13, running this command is not required.

Garden 0.13 should mostly be backwards compatible with your old module configuration. If you find an issue that is not listed in the breaking changes in this page https://docs.garden.io/tutorials/migrating-to-bonsai please report an issue on github.

**Comment by stefreak** [comment]
Sidenote: `garden migrate` fails with the same error message in `0.12.56` with the minimal project config in https://github.com/garden-io/garden/issues/4450#issuecomment-1564516351

**Comment by stefreak** [comment]
My suggestion would be either updating the command to help migration from modules to actions, or removing it. Removing this command would technically not be a breaking change as 0.11 users can still update to 0.12, run the migration, and then update to 0.13.

**Comment by vvagaytsev** [comment]
@stefreak `garden migrate` was removed in #4488. It will be released in `0.13.1`.

**Comment by stefreak** [comment]
Cool, then this can be closed
